src/Esprima/Esprima.csproj

```diff
@@ -26,12 +26,14 @@
 
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <NoWarn>$(NoWarn);1591</NoWarn>
-    
+
     <DefineConstants Condition="'$(Checked)' == true or $(Configuration) == 'Debug'">$(DefineConstants);LOCATION_ASSERTS</DefineConstants>
-    
+
   </PropertyGroup>
+
   <ItemGroup>
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="all" />
+    <PackageReference Include="System.Memory" Version="4.5.4" />
```

Comment:
1- What's coming from it?
2- Would it be removed if TFMs included netcoreapp3.1 ... net6.0?

Reply:
Basically the AsSpan, might  be good idea to refine targets and remove dep if possible